migration/remove-multiple-b-cases/remove-multiple-b-cases.sql

```diff
@@ -0,0 +1,15 @@
+delete from casesvc.caseevent
+where casefk in (
+  select id from (
+    select c.casepk as id, row_number() over (partition by c.casegroupfk order by c.createddatetime desc) as rnum
+    from casesvc."case" c
+  where sampleunittype='B') as innercase
+where rnum > 1);
+
+delete from casesvc.case
+where id in (
+select id from (
+    select c.id as id, row_number() over (partition by c.casegroupfk order by c.createddatetime desc) as rnum
+    from casesvc."case" c
+where sampleunittype='B') as innercase
+where rnum > 1);
```

Comment:
does this script need an explicit BEGIN TRANSACTION at the start and COMMIT TRANSACTION at the end?